cpp/include/cudf/functions.h

```diff
@@ -55,6 +55,21 @@ gdf_error gdf_nvtx_range_pop();
 gdf_error gdf_count_nonzero_mask(gdf_valid_type const *masks,
                                  gdf_size_type num_rows, gdf_size_type *count);
 
+/**
```

Comment:
gdf_num_bitmask_elements should also be exposed to the public. If you are using the C++ API, you will likely want to need that functionality as well

Reply:
Done. This required making `gdf_count_nonzero_mask` no longer be a `__host__ __device__` function, but it turns out it was never being used in a device code, so this wasn't a problem.